.github/workflows/tests.yml

```diff
@@ -46,6 +49,8 @@ jobs:
         shell: bash
```

Comment:
This step shouldn't always be executed

Reply:
I tried `if: ! startsWith()` which seems to invalidate the entire file.  Removing the space between `!` and `startsWith()` seems to be the solution.